src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/tests/latest/test_servicebus_commands.py

```diff
@@ -719,3 +718,104 @@ def test_sb_migration(self, resource_group):
 
         # Delete Namespace - Premium
         self.cmd('servicebus namespace delete --resource-group {rg} --name {namespacenamepremium}')
+
+    @ResourceGroupPreparer(name_prefix='cli_test_sb_network')
+    def test_sb_network(self, resource_group):
+        self.kwargs.update({
+            'loc': 'westus',
+            'rg': resource_group,
```

Comment:
You don't need this. It is automatically registered by the test infrastructure.

Reply:
can you help me with location,  used 'resource_group' for resource group

Reply:
What's wrong with location?  I'm just saying you don't have to specifically register 'rg' here because the infrastructure does it for you